package-lock.json

```diff
@@ -131,6 +131,12 @@
       "integrity": "sha1-aaI6OtKcrwCX8G7aWbNh7i8GOfY=",
       "dev": true
     },
+    "@types/node": {
```

Comment:
Delete lockfile?

Reply:
The lockfile should be kept. I'm not sure what else you're requesting.
https://docs.npmjs.com/configuring-npm/package-lock-json.html